modules/client/src/client-common/constants.ts

```diff
@@ -12,3 +14,21 @@ export const UNAVAILABLE_PROPOSAL_METADATA: ProposalMetadata = {
   description: "(the proposal metadata is not available)",
   resources: [],
 };
+
+export const LIVE_CONTRACTS: { [K in AvailableNetworks]: Deployment } = {
+  mainnet: {
+    // TODO change to mainnet
+    daoFactory: activeContractsList.goerli.DAOFactory,
+    multisigRepo: "multisig.plugin.dao.eth",
+    adminRepo: "admin.plugin.dao.eth",
+    addresslistVotingRepo: "address-list-voting.plugin.dao.eth",
+    tokenVotingRepo: "token-voting.plugin.dao.eth",
+  },
+  goerli: {
+    daoFactory: activeContractsList.goerli.DAOFactory,
+    multisigRepo: activeContractsList.goerli["multisig-repo"],
```

Comment:
We have been testing with addresses here. It would be good to ensure that mainnet works fine with an ENS string

Reply:
I will test this by creating a couple of daos